issues.go

```diff
@@ -401,6 +401,8 @@ type UpdateIssueOptions struct {
 	AssigneeIDs      []int      `url:"assignee_ids,omitempty" json:"assignee_ids,omitempty"`
 	MilestoneID      *int       `url:"milestone_id,omitempty" json:"milestone_id,omitempty"`
 	Labels           *Labels    `url:"labels,comma,omitempty" json:"labels,omitempty"`
+	AddLabels        Labels     `url:"labels,comma,omitempty" json:"add_labels,omitempty"`
+	RemoveLabels     Labels     `url:"labels,comma,omitempty" json:"remove_labels,omitempty"`
```

Comment:
These fields should be pointers and the tags are wrong...

Reply:
@svanharmelen I'll make the edits.

Reply:
Thanks @DavidMBusey! But I just fixed it so we can merge this one 👍